nullaway/src/test/java/com/uber/nullaway/jspecify/GenericMethodTests.java

```diff
@@ -1299,7 +1299,7 @@ public void inferenceFromDataflow() {
             "    // to ensure that dataflow runs",
             "    Object x = new Object(); x.toString();",
             "    Object y = null;",
-            "    // BUG: Diagnostic contains: returning @Nullable expression from method with @NonNull return type",
+            "    // BUG: Diagnostic contains: passing @Nullable parameter 'y' where @NonNull is required",
```

Comment:
This error message now makes sense at least if you know NullAway internals.  Inference for the `id(y)` call fails, so we fall back on treating its parameter and return types as non-null.  So, we get an error that we can't pass in `y`.